governance/program/src/state/realm.rs

```diff
@@ -177,6 +175,34 @@ impl RealmV2 {
         Err(GovernanceError::InvalidGoverningTokenMint.into())
     }
 
+    /// Returns the governing token mint which is used to vote on a proposal given the provided Vote and voting_token_mint
+    ///
+    /// The Veto vote is cast on a proposal configured for the opposite voting population defined using governing_token_mint
+    /// Council can veto Community vote and Community can veto Council assuming the veto for the voting population is enabled
+    ///
+    /// For all votes other than Veto the voting_token_mint is the same as Proposal governing_token_mint
+    pub fn get_proposal_governing_token_mint_for_vote(
+        &self,
+        vote: &Vote,
+        voting_token_mint: &Pubkey,
+    ) -> Result<Pubkey, ProgramError> {
+        if *vote != Vote::Veto {
+            return Ok(*voting_token_mint);
+        }
+
+        // When Community veto Council proposal then return council_token_mint as the Proposal governing_token_mint
+        if self.community_mint == *voting_token_mint {
+            return Ok(self.config.council_mint.unwrap());
+        }
```

Comment:
But this isn't supported yet, correct?

Reply:
Correct. This code was unreachable but better to throw an error here. I fixed it.